### DISCUSSION TITLE
Spawn threads for speed

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#3971] [Enhancement Queue]
via ~~https://twitter.com/zachleat/status/961307855923138560~~ https://www.zachleat.com/twitter/961307855923138560

**Comment by zachleat** [comment]
This repository is now using lodash style issue management for enhancements (see https://twitter.com/samselikoff/status/991395669016436736)

This means enhancement issues will now be closed instead of leaving them open. The enhancement backlog can be found here: https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+

**Comment by chrisdmacrae** [comment]
@zachleat check out [`jest-worker`](https://www.npmjs.com/package/jest-worker). Creates a promise-based multi-threaded worker environment that feels like a regular require/import if you use `async` and `await`.

**Comment by zachleat** [comment]
@chrisdmacrae you follow gatsby stuff too? 😃 I was just looking at this too! 👀

**Comment by zachleat** [comment]
Related to #3585
